setup.py

```diff
@@ -56,12 +56,13 @@
     packages=setuptools.find_packages(),
     install_requires=[
         "jupyter_server>=1.6,<2",
-        "requests~=2.25.0",
-        "peewee~=3.14.0",
-        "jsonschema~=3.2.0",
-        "psutil~=5.8.0",
-        "rucio-clients>=1.26.0",
-        "pyjwt"
+        "requests",
+        "peewee",
+        "jsonschema",
+        "psutil",
+        "rucio-clients>=32.0",
```

Comment:
small silly comment, rucio clients version should be matched to rucio (daemons and server (an UI ideally) versions) for full compatibility - although this it isn't strictly super necessary